packages/blast-utilities/contract-utils.js

```diff
@@ -27,9 +27,9 @@ module.exports.CudosContract = class CudosContract {
     }
   }
 
-  async deploy(initMsg, label = this.#contractLabel) {
+  async deploy(initMsg, funds, label = this.#contractLabel) {
```

Comment:
In order not to introduce breaking change, can you put funds as the last param?

Reply:
if funds are last in line and one wants to set them he will be forced to provide a label as well